training/general/src/environments/acpt-pytorch-1.12-cuda11.6/spec.yaml

```diff
@@ -1,7 +1,8 @@
 $schema: https://azuremlschemas.azureedge.net/latest/environment.schema.json
 
 description: >-
-  Recommended environment in public preview for Deep Learning with PyTorch on Azure containing the Azure ML SDK with the latest compatible versions of Ubuntu, Python, PyTorch, CUDA\RocM, combined with optimizers like ORT Training,+DeepSpeed+MSCCL+ORT MoE and more.
+  Recommended environment for Deep Learning in public preview with PyTorch on Azure containing the Azure ML SDK with the latest compatible versions of Ubuntu, Python, PyTorch, CUDA\RocM, combined with optimizers like ORT Training,+DeepSpeed+MSCCL+ORT MoE and more. The image introduces newly released PyTorch 2.0 for early testing, and preview of new fastcheckpointing capability called Nebula. 
```

Comment:
This statement is not for this image: "The image introduces newly released PyTorch 2.0 for early testing". The same as other images but acpt-pytorch-2.0-cuda11.7.

Is this description supposed to be the same for all images?